Customer_Application/Customer_Api/Controllers/CustomerDataController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Customer_Api.Infrastructure;
+using Customer_Api.Model;
+using Dapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data.SqlClient;
+using UserData;
+
+namespace Customer_Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomerDataController : ControllerBase
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IDataBaseOperation _DataBaseOperation;
+        private readonly ILogger<CustomerDataController> _logger;
+        private readonly IMessageSession session;
+
+
+        public CustomerDataController(IConfiguration configuration, IDataBaseOperation DataBaseOperation, ILogger<CustomerDataController> logger, IMessageSession session)
```

Comment:
CustomerController

Reply:
Change controller name To CustomerController